src/plug.ts

```diff
@@ -280,6 +280,12 @@ export class GlobalPlug implements Plug {
         return this.sdk.evaluate(expression, options);
     }
 
+    public test(expression: string, options: EvaluationOptions = {}): Promise<boolean> {
+        return this.sdk.evaluate(expression, options)
+            .then(result => result === true)
+            .catch(() => false);
```

Comment:
I get the reasoning here of treating runtime evaluation errors as false, like `user's preference's shopPattern is "eager"`.
But I don't think we should hide authentication errors at the plug level, those should be propagated and it's the user's responsibility to handle them

Reply:
Agreed :)